src/test/scala/cromwell/webservice/CromwellApiServiceIntegrationSpec.scala

```diff
@@ -11,8 +12,8 @@ import spray.json._
 import spray.testkit.ScalatestRouteTest
 
 class CromwellApiServiceIntegrationSpec extends FlatSpec with CromwellApiService with ScalatestRouteTest with Matchers {
-  def actorRefFactory = system
-  val workflowManager = TestActorRef(new WorkflowManagerActor(new LocalBackend))
+  def actorRefFactory = new TestWorkflowManagerSystem().actorSystem
+  val workflowManager = TestActorRef(new WorkflowManagerActor(new LocalBackend(system)))
```

Comment:
I'm not entirely sure what `actorRefFactory` is providing here, it's called in `CromwellApiService` as:

```scala
implicit def executionContext = actorRefFactory.dispatcher
```

With a default implementation of

```scala
def actorRefFactory = context
```

I guess I don't quite understand why the implementation was changed in this subclass